core/store/src/trie/ops/resharding.rs

```diff
@@ -53,7 +53,8 @@ fn boundary_account_to_intervals(
             col::DELAYED_RECEIPT_OR_INDICES
             | col::PROMISE_YIELD_INDICES
             | col::PROMISE_YIELD_TIMEOUT
-            | col::BANDWIDTH_SCHEDULER_STATE => {
+            | col::BANDWIDTH_SCHEDULER_STATE
+            | col::GLOBAL_CONTRACT_CODE => {
```

Comment:
I wonder if you hit the `// Poor man's exhaustive check for handling all column types.` 😛

Reply:
that is exactly what happened 😆